lib/datahub-client/data_platform_catalogue/entities.py

```diff
@@ -28,9 +35,22 @@ class ColumnRef(BaseModel):
     A reference to a column in a table
     """
 
-    name: str = Field(description="The column name as it appears in the table")
-    display_name: str = Field(description="A user-friendly version of the name")
-    table: EntityRef = Field(description="Reference to the table the column belongs to")
+    name: str = Field(
+        description="The column name as it appears in the table",
```

Comment:
I'm not sure this description/example are accurate any more - I think this is set to a field path so it would be `table_name.custody_id`

(It should be consistent with the `Column` class)